apps/src/templates/currentUserRedux.js

```diff
@@ -134,7 +135,11 @@ export default function currentUser(state = initialState, action) {
   }
   if (action.type === SET_INITIAL_DATA) {
     const {id, username, user_type, mute_music, under_13} = action.serverUser;
-    analyticsReport.setUserProperties(id, user_type, !!id);
```

Comment:
We can determine signInState just as easily from the `setUserProperties` function so I removed this param. We also probably don't need to log that property anyway but I didn't want to make that change in this PR